### DISCUSSION TITLE
Add a spreadsheet input

### DISCUSSION THREAD

**Discussion by Rednas83** [formkit/formkit#1148] [Ideas]
Add support for a spreadsheet input with options for sorting, filtering and pagination

**Something like this but then with formkit inputs**
![image](https://github.com/formkit/formkit/assets/25219229/a8a22adc-9ef4-49bb-881d-292e16ca7e71)

**Comment by fenilli** [comment]
This is more inline with an UI library not a form one, when would this be added to a form, and what kinda of information does it provide/change?

**Reply by Rednas83** [reply]
The table is the form just like with the form and group type.

Instead off table perhaps spreadsheet is a better name🤔

Is a spreadsheet/table also not some kind of a form?

In case you need some inspiration
https://github.com/linmasahiro/vue3-table-lite

I think it would make a really nice addition

**Reply by fenilli** [reply]
A spreadsheet yes, a table no.

Not sure if its on the list but it might be a good addition to have a spreadsheet.

**Reply by luan-nk-nguyen** [reply]
Hi @Rednas83 — yes we've considered a spreadsheet input before and even built a proof of concept! It's a big one, though, and further down our priority list. Thanks for bringing it up. I renamed it. If there's enough interest we'd consider moving it up the list.

**Comment by Rednas83** [comment]
I have been playing around with [`nuxt ui`](https://ui.nuxt.com/getting-started) lately and it's very cool.
https://ui.nuxt.com/components/table#sortable

Perhaps an idea to add native support for the nuxt ui components via a plugin?
